tools/apigee-envoy-quickstart/extension/README.md

```diff
@@ -0,0 +1,49 @@
+# Apigee Envoy extension (for external access)
+
+This is an extension to the starter setup of apigee-envoy deployment within GKE container platform with Apigee X/Hybrid as API management platform. 
+
+The starter kit deploys httpbin service configured with Envoy proxies as side car proxies. The Envoy proxy is enabled with apigee-adapter as step in the request path enforcing  Apigee provided authentication methods.
+
+This extension enables the expose of deployed sample application (httpbin) externally via istio-ingressgateway. It showcases apigee enabled envoy proxies can offer protection of api traffic intiated externally. 
```

Comment:
* the expose => the exposure?
* apigee => Apigee
* api => API
* ? externally => outside the mesh